uint32 id
float32 u          # Raw pixel u
float32 v          # Raw pixel v
float32 velocity_x # Pixel velocity x
float32 velocity_y # Pixel velocity y
float32 u_norm     # Normalized (undistorted) x: (u - cx) / fx
float32 v_norm     # Normalized (undistorted) y: (v - cy) / fy
